pkg/global/nucleusconstants.go

```diff
@@ -27,15 +27,11 @@ var FrameworkRunnerMap = map[string]string{
 	"jest":    "./node_modules/.bin/jest-runner",
 }
 
-// RawContentURLMap is map of git provider with there raw content url
-var RawContentURLMap = map[string]string{
```

Comment:
why this is removed . We used this to download .tas file .

Reply:
It was also not being used anywhere